include/ddc/kernels/splines/greville_interpolation_points.hpp

```diff
@@ -9,16 +9,28 @@
 #include "spline_boundary_conditions.hpp"
 
 namespace ddc {
+
+template <class Sampling>
+struct UniformSamplingHidden
```

Comment:
why not make it a (private) member type of the class?

Reply:
Isn't there a better name?

Reply:
Usually one cannot use `private` member types due to `nvcc` limitations. But `GrevilleInterpolationPoints` do not seem to initialize the hidden dimension, it is only a local and temporary dimension. I can try to make it a member, like
```
class GrevilleInterpolationPoints
{
  template <class Sampling>
  struct TemporaryUniformSampling {};
}
```

Reply:
I changed the name and made it private